docs/contributing/index.md

```diff
@@ -93,14 +93,28 @@ A package configures the types it allows in a file `towncrier.toml` located at t
 The content of this file must include the following.
 
 -   A brief message that summarizes the changes in your contribution.
+    -   Use a narrative format, in the past tense, proper English spelling and grammar, complete sentences, and inline markup as needed.
```

Comment:
We don't actually use complete sentences (including in your example below); usually the subject is implied, e.g. "Added X." or "Updated Y."